src/paperless/settings.py

```diff
@@ -921,7 +921,7 @@ def _parse_cachalot_settings():
     ttl = __get_int("PAPERLESS_READ_CACHE_TTL", 3600)
     ttl = min(ttl, 31536000) if ttl > 0 else 3600
     _, redis_url = _parse_redis_url(
-        os.getenv("PAPERLESS_READ_CACHE_REDIS_URL", None),
+        os.getenv("PAPERLESS_READ_CACHE_REDIS_URL", os.getenv("PAPERLESS_REDIS", None)),
```

Comment:
```suggestion
        os.getenv("PAPERLESS_READ_CACHE_REDIS_URL", _CHANNELS_REDIS_URL),
```

Since we already do

```
_CELERY_REDIS_URL, _CHANNELS_REDIS_URL = _parse_redis_url(
    os.getenv("PAPERLESS_REDIS", None),
)
```